RFS/src/main/java/org/opensearch/migrations/bulkload/common/OpenSearchClient.java

```diff
@@ -312,7 +312,7 @@ public Optional<ObjectNode> getSnapshotStatus(
 
     protected abstract String getBulkRequestPath(String indexName);
 
-    Retry getBulkRetryStrategy() {
+    protected Retry getBulkRetryStrategy() {
```

Comment:
This was the source of the mocktio about stubbings being incomplete, I'm going to look into this a little further, because this was a very strange fix - unclear to the 'why' but this should get us unblocked.

```
org.mockito.exceptions.misusing.UnfinishedStubbingException: 
Unfinished stubbing detected here:
-> at org.opensearch.migrations.bulkload.common.OpenSearchClientTest.testBulkRequest_record…
```